source/upgrade/important-upgrade-notes.rst

```diff
@@ -17,534 +17,540 @@ Important Upgrade Notes
    - Support for Mattermost Server v5.31 `Extended Support Release <https://docs.mattermost.com/upgrade/extended-support-release.html>`_ has come to the end of its life cycle as of October 15, 2021. Upgrading to Mattermost Server v5.37 `Extended Support Release <https://docs.mattermost.com/upgrade/extended-support-release.html>`_ or later is required.
    - The "version" CLI command will be deprecated in a future release.
    - Upgrading Mattermost can result in an error “ERROR: default for column "column_name" cannot be cast automatically to type jsonb” if you have non-JSON data in a column. This can also happen if the database data has been manipulated by external processes (e.g. plugins) and in cases where improper input data could get silently truncated due to varchar limits. The data in the affected database column has to be manually fixed.
+   - Channel results in the channel autocomplete will include private channels. Customers using `Bleve <https://docs.mattermost.com/deploy/bleve-search.html>`_ or `Elasticsearch <https://docs.mattermost.com/scale/elasticsearch.html>`_ for autocomplete will have to reindex their data to get the new results. Since this can take a long time, we suggest disabling autocomplete and running indexing in the background. When this is complete, re-enable autocomplete. Note that only channel members will see private channel names in autocomplete results.
```

Comment:
This can be moved to the v6.2 section. Sorry, I meant that the CRT known issue can be moved up to the bullet points since it affects versions 5.37+.